pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -13,7 +13,7 @@
 # function correctly.
 rustPlatform.buildRustPackage rec {
   pname = "prisma-engines";
-  version = "4.6.0";
+  version = "4.6.1";
 
   src = fetchFromGitHub {
```

Comment:
Shall we also update sha256 here?

Reply:
Do you mean the hash on line 22? I think it doesn't need to be updated because the Cargo dependencies are exactly the same as in 4.6.0.

Reply:
@tomhoule When I tested your change locally, I still trigger the same `enum` bug that was fixed in 4.6.1. When I [updated the sha](https://github.com/sethmiller/nixpkgs/commit/32f1d6b441e302b75a90e787f54262c540ae8c66) and retested, I could no longer reproduce the issue. It was odd because `prisma version` reported `4.6.1` but it very much appears that `4.6.0` was being executed. I don't have a great understanding of the Nix internals, admittedly, but perhaps the sha is used for a cache look up but the cache results aren't validated against the other `fetchFromGithub` parameters. Or I just did something wrong when testing.

Reply:
Hmm that's interesting and confusing :/ 4.7.0 was just released today, let's explore that hash issue there.

Reply:
Yeah, I don't get it either. My working theory is that I did something wrong but it seems like if I set up test wrong it be consistently wrong. I tested with 4.5.0, this PR, and my 4.6.1 change. 4.50 and mine worked, this PR didn't.

This was my test script ran with `nix-shell`

```nix
with (import <unstable> {});
let
     prisma_4_6_1 = import (builtins.fetchGit {
         # Descriptive name to make the store path easier to identify
         name = "my-old-revision";
         url = "https://github.com/sethmiller/nixpkgs/";
         ref = "refs/heads/prisma-4-6-1";
         rev = "32f1d6b441e302b75a90e787f54262c540ae8c66"; # My commit sha
     }) {};

     pe = prisma_4_6_1.prisma-engines;
in
mkShell {
  buildInputs = [
    nodejs_latest
    pe
  ];
  shellHook = ''
      mkdir -p .nix-node
      export NODE_PATH=$PWD/.nix-node
      export NPM_CONFIG_PREFIX=$PWD/.nix-node
      export PATH=$NODE_PATH/bin:$PATH
      export PRISMA_MIGRATION_ENGINE_BINARY="${pe}/bin/migration-engine"
      export PRISMA_QUERY_ENGINE_BINARY="${pe}/bin/query-engine"
      export PRISMA_INTROSPECTION_ENGINE_BINARY="${pe}/bin/introspection-engine"
      export PRISMA_FMT_BINARY="${pe}/bin/prisma-fmt"
      export PRISMA_QUERY_ENGINE_LIBRARY="${pe}/lib/libquery_engine.node"
  '';
}
```